src/common/tests/btree_direct.c

```diff
@@ -519,8 +545,8 @@ sk_btr_kv_operate(enum sk_btr_opc opc, char *str, bool verbose)
 			daos_iov_set(&val_iov, NULL, 0); /* get address */
 			rc = dbtree_lookup(sk_toh, &key_iov, &val_iov);
 			if (rc != 0) {
-				D_ERROR("Failed to lookup %s\n", key);
-				return -1;
+				sprintf(outbuf, "Failed to lookup %s\n", key);
```

Comment:
Move this to standard utils so can be called with single line in current file and can be used in other unit test files.

Reply:
As discussed this may have to rethink this method. The input for the function should handle %s, %s, %u or DF_64U types. We will not implement this change for now.